### ISSUE TITLE
<!DOCTYPE html>

### ISSUE THREAD

**Issue by xiaoyueyue165** [eslint/eslint#9669]
**Tell us about your environment** vscode 使用的Eslint插件

* **ESLint Version:**1.4.3
* **Node Version:** 8.4
* **npm Version:**

**What parser (default, Babel-ESLint, etc.) are you using?**
 etc

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

   module.exports = {
    "env": {
        "browser": true,
        "es6": true,
        "node":true
    },
    // "extends": "eslint:recommended",//默认规则
    "extends": "airbnb",
    "parserOptions": {
        "ecmaFeatures": {
            "experimentalObjectRestSpread": true,
            "jsx": true
        },
        "sourceType": "module"
    },
    "plugins": [
        "react"
    ],
    "rules": {
        "indent": [
            "off",
            "tab"
        ],
        "linebreak-style": [
            "off", 
            "windows" //关掉换行报错 
        ],
        "quotes": [
            "off",
            "double" //字符串首先要用""包含起来 
        ],
        "semi": [
            "error",
            "always" //”;”一直要记得加 
        ],
        "no-console": 0, //允许使用console 
        // "eqeqeq": 2, //使用===和!== 
        "no-else-return": 2, //if语句中不准在return之后使用else 
        "no-empty-function": 2,//禁止出现空函数 
        "no-cond-assign": "error", //在条件判断中不能出现赋值语句
        "prefer-arrow-callback": "warn"//ES6的箭头回调函数标记法
    }
};

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
 <!DOCTYPE html>
<head>  
  <title>React</title>
  <script type="text/javascript" crossorigin src="https://unpkg.com/react@16/umd/react.development.js"> </script>
  <script type="text/javascript" crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"> </script>
  <script type="text/javascript" src="https://cdn.bootcss.com/babel-core/5.8.24/browser.js"> </script>

</head>

<body>
  
  <div id="root"></div>
  <script type="text/babel">

class Toggle extends React.Component {
  constructor(props) {
    super(props);
    this.state = {isToggleOn: true};

    // This binding is necessary to make `this` work in the callback
    this.handleClick = this.handleClick.bind(this);
  }

  handleClick() {
    this.setState(prevState => ({
      isToggleOn: !prevState.isToggleOn
    }));
  }

  render() {
    return (
      <button onClick={this.handleClick}>
        {this.state.isToggleOn ? 'ON' : 'OFF'}
        
      </button>
      
    );
  }
}

ReactDOM.render(
  
  <Toggle />,
  document.getElementById('root')
);
  </script>
</body>
</html>





```

```bash
  eslint onOff.html

```

**What did you expect to happen?**
 Each Html file is incorrect，Located at`<!DOCTYPE html>`I am just getting started, do not know how to solve the problem, please help me

**What actually happened? Please include the actual, raw output from ESLint.**
  1:2  error  Parsing error: Unexpected token !

✖ 1 problem (1 error, 0 warnings)

**Comment by platinumazure** [comment]
ESLint does not support parsing non-JavaScript code. You need to use a [processor](https://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins) to extract the JavaScript code out of other files for linting.

I would suggest looking at [eslint-plugin-html](https://github.com/BenoitZugmeyer/eslint-plugin-html/). That should help you get where you need.

**Comment by xiaoyueyue165** [comment]
It's cool, it's working, thanks

**Comment by ilyavolodin** [comment]
Closing as the question has been answered.
